docs/docs/overview/quickstart/get-started-with-dotnet.md

```diff
@@ -0,0 +1,359 @@
+---
+sidebar_position: 11
```

Comment:
Please check this number again to verify whether there are other quickstart guides that have it.

Reply:
yup, this puts it at the bottom of the list

Reply:
okay